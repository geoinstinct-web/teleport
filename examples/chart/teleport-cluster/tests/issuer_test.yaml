suite: Issuer
templates:
  - issuer.yaml
tests:
  - it: do not create Issuer by default
    asserts:
      - hasDocuments:
          count: 0

  - it: create Issuer when highAvailability.certManager.issuerSpec is specified
    values:
      - ../.lint/cert-issuer.yaml
    asserts:
      - hasDocuments:
          count: 1
      - isKind:
          of: Issuer
      - matchSnapshot: {}
