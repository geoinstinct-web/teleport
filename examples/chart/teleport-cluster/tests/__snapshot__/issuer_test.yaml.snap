create Issuer when highAvailability.certManager.issuerSpec is specified:
  1: |
    apiVersion: cert-manager.io
    kind: Issuer
    metadata:
      name: letsencrypt-production
      namespace: NAMESPACE
    spec:
      acme:
        email: email@address.com
        privateKeySecretRef:
          name: letsencrypt-production
        server: https://acme-v02.api.letsencrypt.org/directory
        solvers:
        - dns01:
            route53:
              hostedZoneID: Z0159221358P96JYAUAA4
              region: us-east-1
          selector:
            dnsZones:
            - example.com
